static/src/stylesheets/module/_adslot.scss

```diff
@@ -178,8 +178,13 @@
     .top-banner-ad-container--desktop & {
         margin: 0 auto;
         display: none;
-        min-height: 90 + ($gs-row-height / 2);
+        min-height: 90px;
         padding-bottom: $gs-row-height / 2;
+        padding-top: $gs-row-height / 2;
+
+        .ad-slot__label {
+            margin-top: -$gs-row-height / 2;
```

Comment:
i'd tend to opt for `0 - $gs-row-height` (or similar) for legibility, mainly for times like this, where there are a lot of hyphens in the name and calcs with it 

i.e. spaces around operators, since `$my-value+$other-value - $another-value*$final-value` is valid but not nec clear